lisa/target.py

```diff
@@ -554,6 +567,31 @@ def _init_target(self, kind, name, workdir, device, host,
                 logger.warning('Failed to initialized "{}" devlib Module'.format(module))
         return target
 
+    def _init_kmodules(self, kernel_src, kernel_modules):
+        """
+        Compile and load any modules specified in the config.
+
+        :param kernel_src: Path to kernel source tree to use to compile modules
+        :type name: str
+
+        :param kernel_modules: List of kernel modules paths to build and load
+        :type kernel_modules: StrList
+        """
+        if not kernel_modules:
+            return
+
+        if not kernel_src:
+            raise RuntimeError("Can't build modules without a kernel source tree")
```

Comment:
May be more appropriate to raise a ValueError, since it boils that to the user incorrectly giving a `None` value (explicitly or implicitly) for the use case. RuntimeError is more the "last resort" exception for things that don't fit in any other category:
https://docs.python.org/3/library/exceptions.html#ValueError